src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition-duration: 0.3s;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-color;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__rating-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__price-key {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 400;
+    color: $secondary-color;
+  }
+
+  &__price-value {
```

Comment:
could've used sass nesting because they have same prefix

Reply:
I cant nesting them cos 
&__price-container {
  &-key {
   }
 }
 = &__price-container-key
 
 thats not a bem price block.... 

I can change names from price-key to key and price-value to value. Would it be okey?

Reply:
i think that @denys-danyliuk had in mind the following nesting:
```
&__price {
  &-container {  
    ...  
  }
  &-key {  
    ...  
  }
  &-value {  
    ...  
  }
}
```